resources/helm/v2.1/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -37,22 +37,22 @@ metadata:
     release: {{ .Release.Name }}
 spec:
   hosts:
-  - istiod.{{ .Release.Namespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
+  - istiod-{{ .Values.revision | default "default" }}.{{ .Release.Namespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
```

Comment:
this is correct- but do we need the meshexpansion stuff? we don't support it, right?

Reply:
We don't, but my original tests were using MeshNetworks, so I had to get this working for that.  Didn't want to just throw the changes away.

Reply:
gotcha